spec/features/dark_file_object_spec.rb

```diff
@@ -10,7 +10,7 @@
   let(:bare_druid) { 'tx666tx9999' }
   let(:object_staging_dir) { Rails.root.join(Settings.assembly_staging_dir, 'tx', '666', 'tx', '9999', bare_druid) }
   let(:cocina_model_dark_access) { instance_double(Cocina::Models::Access, access: 'dark') }
-  let(:item) { instance_double(Cocina::Models::DRO, type: Cocina::Models::Vocab.media, access: cocina_model_dark_access) }
+  let(:item) { instance_double(Cocina::Models::DRO, type: Cocina::Models::Vocab.object, access: cocina_model_dark_access) }
```

Comment:
The tests should really use the correct object type, media happened to work, but its not really reflective of the test.  Same for other tests below.